### PR TITLE
[PRODU-252] log to stderr and keep track of event metadata

### PR DESCRIPTION
**Pull request by michielrogissart-datavillage**
[PRODU-252](https://app.plane.so/datavillage/projects/326ee63f-7e0a-47c2-b83b-af85023dd846/issues/08de9fc2-4a47-4557-8225-e5295335bf09)

### CODE REVIEW

File: dv_utils/log_utils.py [hunk 1]
```diff
@@ -5,25 +5,36 @@
...
     ERROR = 40
 
+# Holds metadata about the current event that is added to every log statement
+# Do not create an object of this class in code
+class LogMetadata:
+    def __init__(self):
+        self.evt = None
+        self.evt_received = None
+        self.evt_stream = None
+        self.app_id = default_settings.config("DV_APP_ID", None)
```
Comment (michielrogissart-datavillage) [inline dv_utils/log_utils.py:26]: no evt_id yet, not sure where this is generated and how it is communicated. Maybe separate ticket?

File: dv_utils/listener.py [hunk 2]
```diff
@@ -29,14 +29,15 @@ def __init__(
            if evt:
                start = time.time()
                evt_type =evt.get("type", "MISSING_TYPE")
+               set_event(evt)
                if(log_events):
-                  audit_log("Event processing started", evt=evt_type, state="STARTED", app="algo")
```
Comment (arnegoeteyn) [inline dv_utils/listener.py:33]: Waarom hier die evt weg? Da's toch handige info om te weten welk event start en stopt?

Comment (arnegoeteyn) [reply dv_utils/listener.py:33]: ah ja via die set_event, ik zie het!

File: dv_utils/log_utils.py (hunk 1 above)
Comment (arnegoeteyn) [reply dv_utils/log_utils.py:26]: ok

Comment (arnegoeteyn) [reply dv_utils/log_utils.py:26]: let's plan this in dv_utils module

**Review by arnegoeteyn — APPROVED** [review]
lgtm!

**Pull request merged**